mpu6050/mpu6050.c

```diff
@@ -51,6 +51,16 @@ static int mpu6050_read_data(void)
 	if (temp * 2 > DIV)
 		fract_part_cels++;
 
+	int_part_fahr = (int_part_cels * 9) / 5;
+	temp = (int_part_cels * 9) % 5;
+	fract_part_fahr = (temp * MULTIPL) / 5;
+	fract_part_fahr += ((fract_part_cels * 9) / 5);
+	while (fract_part_fahr > MULTIPL) {
```

Comment:
I think it is possible to implement conversation without while loop and it is better to get F directly from mpu6050 internal representation. You can change the divisor and and offset.

Reply:
@yekovalyov, of course I can remove while loop for this implementation.

I just wanted to extend functionality and print also Celsius value of temperature. Am I need to remove Celsius representation and get only Fahrenheit value directly from internal mpu6050 values?

Reply:
You don't need to remove Celsius value. But it is better to reduce the sequence of operations.